config/initializers/099-early-hints.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "middleware/early_hints"
+
+if ENV['DISCOURSE_EARLY_HINTS']
```

Comment:
I would use global_settings for this, also recommend requiring the file after you know you need it. You still get to use ENV to set it, but it documents the setting far better cause it is in discoruse_defaults